### DISCUSSION TITLE
[QA] SIGN IN

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#267] [Q&A]
sign in

**Comment by hwoo3303** [comment]
로그아웃하면 페이지 새로고침이 필요해보입니다.
상세페이지 외에도 생성/수정 페이지에서도 권한이 없는 사용자가 생성/수정 api를 호출하는 상황이 발생합니다.

https://user-images.githubusercontent.com/15838144/174768828-0617f139-ee5b-4c30-a79a-b71464ccd527.mov

**Reply by NamgyungKim** [reply]
아 header 부분은 제가 수정하겠습니다 감사합니다.~

**Comment by NamgyungKim** [comment]
로그인 상태에서는 해당 페이지 에 접근하지 못하도록해야할 것 같습니다.

**Comment by Hyevvy** [comment]
로그인시 enter로도 가능하게 하면 좋을 거 같아용 !! (그냥 의견입니당 ㅎㅎ ) 

https://user-images.githubusercontent.com/72402747/174770157-45d8bbce-7903-4b1e-af07-40b95e1946e4.mov
